stable_baselines3/common/policies.py

```diff
@@ -559,15 +563,18 @@ def _build(self, lr_schedule: Schedule) -> None:
             # features_extractor/mlp values are
             # originally from openai/baselines (default gains/init_scales).
             module_gains = {
+                self.features_extractor: np.sqrt(2),
```

Comment:
The attributes `self.pi_features_extractor` and `self.vf_features_extractor` are created anyway in the `ActorCriticPolicy`'s constructor, and, in case the features extractor is shared, they are simply aliases for `self.features_extractor`.
So, at this point, wouldn't it be possible to do something like what follows?

```suggestion
                self.pi_features_extractor: np.sqrt(2),
```

Reply:
I removed `del self.features_extractor`, should be the cleaner solution